doc/api/Player_Errors.md

```diff
@@ -382,11 +382,34 @@ the following properties:
 Those errors are linked to the "Encrypted Media Extensions" API. They concern various
 DRM-related problems.
 
-They all have a `type` property equal to `"ENCRYPTED_MEDIA_ERROR"`.
+They have the following properties:
 
-When its code is set to `KEY_STATUS_CHANGE_ERROR`, an ENCRYPTED_MEDIA_ERROR generally also
-have a `keyStatuses` property, which is documented in the corresponding
-`KEY_STATUS_CHANGE_ERROR` code explanation below.
+- `type` (`string`): Equal to `"ENCRYPTED_MEDIA_ERROR"`.
+
+- `keySystem` (`string | undefined`): If known, the
+  [`keySystem`](https://developer.mozilla.org/en-US/docs/Web/API/MediaKeySystemAccess/keySystem)
+  of the `MediaKeySystemAccess` that encountered the error. This might be useful to check
+  e.g. if the error was encountered while relying on PlayReady, Widevine or other
+  indentified technologies.
+
+  Note that it is not set for some errors for which it wouldn't make sense, such as for
+  `EncryptedMediaError` instances with the `code` `MEDIA_IS_ENCRYPTED_ERROR`.
+
+- `mediaKeySystemConfiguration` (`MediaKeySystemAccessConfiguration | undefined`): If
```

Comment:
The key name in the doc is not matching the actual key name

Reply:
? it is ?

Though it may be a bad name as written earlier

Reply:
Actually `mediaKeySystemAccessConfiguration` is the name in the error constructor but it's renamed to `mediaKeySystemConfiguration`. So the doc is correct 

Given that there is a public API method called `getKeySystemConfiguration` I would say that naming it `keySystemConfiguration` seems the more coherent